src/encoded/static/components/typeutils.js

```diff
@@ -296,23 +296,32 @@ export function fileStatusList(session, sessionProperties) {
 
 // Display supersedes/superseded_by lists.
 export const Supersede = ({ context }) => {
-    // Make array of superseded_by accessions
-    let supersededBys = [];
-    if (context.superseded_by && context.superseded_by.length) {
-        supersededBys = context.superseded_by.map(supersededBy => globals.atIdToAccession(supersededBy));
-    }
-
     // Make array of supersedes accessions
     let supersedes = [];
     if (context.supersedes && context.supersedes.length) {
         supersedes = context.supersedes.map(supersede => globals.atIdToAccession(supersede));
     }
 
-    if (supersededBys.length > 0 || supersedes.length > 0) {
+    let supersededByOutput = null;
+    if (context.superseded_by && context.superseded_by.length > 0) {
```

Comment:
This is minor and can be ignored if desired. Lines 306 and 320 repeat the statement "context.superseded_by && context.superseded_by.length > 0". That can be refactored into a variable.

Reply:
I agree with this, and have updated the code.